test/util/hca-helpers.js

```diff
@@ -436,12 +436,25 @@ function completeVaInsuranceInformation(client, data, onlyRequiredFields) {
 }
 
 function initApplicationSubmitMock() {
+  // TODO: Fully migrate to v1
+  mock(null, {
+    path: '/v0/health_care_applications',
+    verb: 'post',
+    value: {
+      data: {
+        formSubmissionId: '123fake-submission-id-567',
+        timeStamp: '2016-05-16'
+      }
+    }
+  });
   mock(null, {
     path: '/api/hca/v1/application',
     verb: 'post',
     value: {
-      formSubmissionId: '123fake-submission-id-567',
-      timeStamp: '2016-05-16'
+      data: {
```

Comment:
I think for this old endpoint we don't want this information to be nested within `data`, at least it was working find that way before.